doc/decorators.md

```diff
@@ -42,7 +42,17 @@ class HelloWorld {}
 @resource('i-want-bacon')
 class HelloWorld {}
 ```
+## @identifier()
 
+This decorator comes in handy when you have the same resource name for multiple endpoints. Without it, aurelia-orm will use the resource decorator. The identifier will be used to register and retrieve the repository for the given entity.
+
+When left empty, the name of the class (.toLowerCase()) will be used as the resource name. This is usually fine.
```

Comment:
Maybe add a note that this is a bad idea for bundling, as the names of classes get renamed.